app/models/services/crossref/resource.rb

```diff
@@ -0,0 +1,52 @@
+class Crossref
+  class Resource
+    attr_accessor :id, :response
+
+    def initialize id
+      self.id = id.to_s
+      endpoint = URI.parse("http://api.crossref.org/works/#{self.id}")
```

Comment:
Oh, I see what you mean now. Good point. I think I need it more in the `Pubmed::Resource` class. Good catch